tools/replay-archive/src/cli.rs

```diff
@@ -313,11 +314,13 @@ impl ReplayController {
                 prev_chunk_header.height_included(),
             )?;
 
+            let transactions = SignedValidPeriodTransactions::new(
+                chunk.to_transactions().to_vec(),
+                vec![true; chunk.to_transactions().len()],
```

Comment:
Looks like `// FIXME: see the `validate_chunk` thing above.` should still stay, because it is not guaranteed that all transactions are valid.